media/livestream/api/LiveStream.Samples/GetInput.cs

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START livestream_get_input]
+
+using Google.Cloud.Video.LiveStream.V1;
+
+public class GetInputSample
+{
+    public Input GetInput(
+         string projectId, string locationId, string inputId)
+    {
+
```

Comment:
nit: remove blank line, I've seen it in a couple other samples as well

Reply:
Done